tgui/packages/tgui/interfaces/StripMenu.tsx

```diff
@@ -213,6 +213,13 @@ const SLOTS: Record<
     gridSpot: getGridSpotKey([4, 5]),
     image: 'inventory-pocket.png',
   },
+  // Doppler station addition start
+  tail: {
+    displayName: 'tail',
+    gridSpot: getGridSpotKey([3, 3]),
+    image: 'inventory-belt.png',
+  },
+  // Doppler station addition end
```

Comment:
```suggestion
  { /* Doppler station addition start */ }
  tail: {
    displayName: 'tail',
    gridSpot: getGridSpotKey([3, 3]),
    image: 'inventory-belt.png',
  },
  {/* Doppler station addition end */ }
```

UI comments can be fickle because UI files gets minified having just // comments can break things